gobblin-compaction/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
   runtime externalDependency.datanucleusCore
   runtime externalDependency.datanucleusRdbms
 
+  compileOnly externalDependency.orcMapreduce
```

Comment:
Is this here because of upstream changes?

Reply:
Yea, this change is not quite related to the objective of this PR. intellij does not find a few classes if we do not add this dependency.